qemu/tests/rng_bat.py

```diff
@@ -47,61 +49,44 @@ def set_winutils_letter(cmd, session, params):
             vol = "%s:" % vol
         return cmd.replace("X:", vol)
 
-    driver_name = params.get("driver_name")
     rng_data_rex = params.get("rng_data_rex", r".*")
     dev_file = params.get("filename_passthrough")
     timeout = float(params.get("login_timeout", 360))
     rng_dll_register_cmd = params.get("rng_dll_register_cmd")
     read_rng_timeout = float(params.get("read_rng_timeout", "360"))
     cmd_timeout = float(params.get("session_cmd_timeout", "360"))
-
-    error.context("Boot guest with virtio-rng device", logging.info)
+    error_context.context("Boot guest with virtio-rng device", logging.info)
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     vm_pid = vm.get_pid()
     session = vm.wait_for_login(timeout=timeout)
-    session_cmds_key = ["read_rng_cmd",
-                        "enable_verifier_cmd",
-                        "driver_verifier_cmd"]
-    session_cmds = (set_winutils_letter(params.get(key), session, params)
-                    for key in session_cmds_key)
-    read_rng_cmd, enable_verifier_cmd, driver_verifier_cmd = session_cmds
 
     if dev_file:
-        error.context("Check '%s' used by qemu" % dev_file, logging.info)
+        error_context.context("Check '%s' used by qemu" % dev_file, logging.info)
         if not is_dev_used_by_qemu(dev_file, vm_pid):
```

Comment:
according to the case RHEL7-11030, need to check rng pci device and hardware RNG core before reading data from  /dev/random in linux guest. shall we need to add the steps?